docusaurus/docs/iOS/swiftui/components/typing-indicators.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Typing Indicators
+---
+
+## Typing Indicators Overview
+
+The SwiftUI SDK has support for typing indicators, shown when other participants in a conversation are typing. There are two places where the typing indicators are shown. The first one is when the user is in the channels list - when someone in a channel is typing, the indicator is shown in the corresponding channel list item. The second place, where a typing indicator is shown, is in the chat channel view. Depending on the configuration provided, the typing indicator can be shown either in the navigation bar, or above the composer, as an overlay over the message list. 
+
+The configuration for placing the typing indicator can be found in the `TypingIndicatorPlacement` enum, which is part of the `Utils`  class. By default, the placement of the typing indicator is above the composer, which is represented by the enum value `bottomOverlay`.
+
+Here's an example how to change the configuration, so that the typing indicator is shown in the navigation bar.
```

Comment:
Suggestion:

```suggestion
Here's an example of how to change the configuration, so that the typing indicator is shown in the navigation bar (represented by the `TypingIndicatorPlacement` enum value `.navigationBar`).
```